storage/StorageExample/Shared/ViewModel.swift

```diff
@@ -15,20 +15,29 @@
 //
 
 import Foundation
-import UIKit
 import SwiftUI
+#if !os(macOS)
+  import UIKit
+#endif
 
+@MainActor
 class ViewModel: ObservableObject {
   @Published var image: Image?
   @Published var showingImagePicker = false
-  @Published var inputImage: UIImage?
+
+  #if os(iOS)
+    @Published var inputImage: UIImage?
+  #elseif os(macOS)
+    @Published var inputImage: NSImage?
```

Comment:
Do you need to `import AppKit` if `os(macOS)`?

Reply:
I think `SwiftUI` is sufficient to handle the UI. Without importing `AppKit`, the app can be built and run successfully.